R/fct_SQLite.R

```diff
@@ -325,13 +335,11 @@ db_save_review <- function(
     dplyr::inner_join(dplyr::tbl(db_con, "row_ids"), by = review_by) |> 
     # Filter below prevents unnecessarily overwriting the review status in forms   
     # with mixed reviewed status (due to an edit by the investigators). 
-    dplyr::filter(reviewed != new_review_state) |> 
```

Comment:
This can be removed? Thus, identification of which rows within a form needs an update is completely done in the SQL statement in this function now? I will double check later.

Reply:
I did not realize this was part of the review process until reviewing your latest PR fixing the errant error message. I actually fixed it the follow-up PR. I will add this line back in here.

Reply:
Added back in